src/md/src/InstallationSteps/SetupSSL.md

```diff
@@ -1,139 +1,53 @@
-         - RP_SESSION_LIVE=86400 #in seconds
-       restart: always
+- Server with a public IP address, with Docker and docker-compose installed on it
+- Installed ReportPortal on this servers
+- Your own domain and the DNS configured accordingly so the hostname records
 
-     gateway:
-       image: fabiolb/fabio:1.5.2-go1.8.3
-       ports:
-         - "9998:9998" # GUI/management
-         - "8080:9999" # HTTP exposed
-       volumes:
-         - ./certs:/certs
-       environment:
-         - FABIO_REGISTRY_CONSUL_ADDR=registry:8500
-         - FABIO_REGISTRY_CONSUL_REGISTER_NAME=gateway
-         - FABIO_PROXY_ADDR=:9999;cs=av;rt=300s;wt=300s
-         - FABIO_PROXY_CS=cs=av;type=file;cert=/certs/cert.pem;key=/certs/mykey.pem
+### Traefik configuration
 
-     index:
-       image: reportportal/service-index:3.1.0
-       environment:
-         - RP_SERVER.PORT=8080
-         - RP_CONSUL.TAGS=urlprefix-/
-         - RP_CONSUL.ADDRESS=registry:8500
-       depends_on:
-          - registry
-          - gateway
-       restart: always
-
-     api:
-       image: reportportal/service-api:3.1.1
-       depends_on:
-         - redis
-         - mongodb
-       environment:
-         - RP_PROFILES=docker
-         - RP_ISSUE_ANALYZER_DEPTH=4
-         - JAVA_OPTS=-Xmx1g -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp
-       restart: always
-
-     ui:
-       image: reportportal/service-ui:3.1.5
-       environment:
-         - RP_SERVER.PORT=8080
-         - RP_CONSUL.TAGS=urlprefix-/ui opts strip=/ui
-         - RP_CONSUL.ADDRESS=registry:8500
-       restart: always
-   ```
+Provided below is an example of using Traefik (gateway service) in docker-compose.yml. If you don't have any custom configurations, you are free to use the example below.
 
-#### Adding certificates to the server
-Open up the ReportPortal directory (place, where is the "data" folder presents). Create the directory
-"certs". Add cert "cert.pem" and key "mykey.pem" to this directory.
+Please do not forget to update variables acme.email and acme.domains with your own values
 
-#### Recreate containers
-Run
 ```$xslt
-docker-compose -p reportportal up -d --force-recreate
-```
-Server part is ready.
-To verify result open up report-portal via https connection.
-
-### Project configuration
-#### Adding certificate to project
-Before adding certificates to the project, you need to convert it from "PEM" format to "DER"*.
-Find any online converter or use line below to do it:
-```$xslt
-openssl x509 -outform der -in cert.pem -out cert.der
-```
-After the conversion is completed, you need to add the converted certificate to the jks container.
-To do it, open up the bin folder in your JAVA JDK directory and run:
-```$xslt
-keytool -import -alias your-alias -keystore certstore.jks -file cert.der
-```
-NB! Remember your password, after adding it to container.<br>
-Add the certstore.jks file to resources folder in the classpath of your project.
+gateway:
+    image: traefik:1.6
```

Comment:
1.7?

Reply:
@AlehB ^

Reply:
@avarabyeu , there is traefik:1.6 in the latest docker-compose.yml provided by @pbortnik